src/main/java/bio/terra/workspace/app/controller/TemplateApiController.java

```diff
@@ -0,0 +1,21 @@
+// package bio.terra.workspace.app.controller;
```

Comment:
still needed?

Reply:
Looks like it was only PingService, deleted it